plugins/DataLoadRlog/dataload_rlog.cpp

```diff
@@ -54,6 +54,18 @@ const std::vector<const char*>& DataLoadRlog::compatibleFileExtensions() const
 
 bool DataLoadRlog::readDataFromFile(FileLoadInfo* fileload_info, PlotDataMapRef& plot_data)
 {
+  QStringList dbc_items;
```

Comment:
Only show DBC dialog when there is `can` data in the rlog. Is it possible to defer the dialog to the point where you instantiate the parser?

Reply:
If the env variable `DBC_NAME` is set you can use that instead of the dialog.